test/es-module/test-esm-loader-chaining.mjs

```diff
@@ -1,7 +1,9 @@
-import '../common/index.mjs';
-import fixtures from '../common/fixtures.js';
+import { spawnPromisified } from '../common/index.mjs';
+import { fileURL } from '../common/fixtures.mjs';
```

Comment:
nit: I find `fixtures.fileURL(` much more parsable than `fileURL(`.
```suggestion
import * as fixtures from '../common/fixtures.mjs';
```

Reply:
meh, seems the same to me, but if it makes you happy ;)